auction-server/src/traced_sender_svm.rs

```diff
@@ -0,0 +1,75 @@
+use {
+    crate::config::ChainId,
+    axum::async_trait,
+    axum_prometheus::metrics,
+    solana_client::{
+        client_error,
+        nonblocking::rpc_client::RpcClient,
+        rpc_client::RpcClientConfig,
+        rpc_request::RpcRequest,
+        rpc_sender::{
+            RpcSender,
+            RpcTransportStats,
+        },
+    },
+    solana_rpc_client::http_sender::HttpSender,
+    std::time::{
+        Duration,
+        Instant,
+    },
+};
+
+pub struct TracedSenderSvm {
```

Comment:
I think `TracedClientSvm` is a better name since we do all RPC interactions through this and it's not just for sending.